src/System.Runtime.InteropServices/src/System.Runtime.InteropServices.csproj

```diff
@@ -44,15 +44,13 @@
   </ItemGroup>
   <ItemGroup>
     <ReferenceFromRuntime Include="System.Private.CoreLib" />
+    <ProjectReference Include="..\..\System.Runtime.Extensions\src\System.Runtime.Extensions.csproj"/>
```

Comment:
I believe this should just work, but if it doesn't, try something like:
```xml
<ProjectReference Include="..\..\System.Runtime.Extensions\src\System.Runtime.Extensions.csproj">
   <TargetGroup>$(TargetGroup)</TargetGroup>
</ProjectReference>
```

I feel like that won't and shouldn't be necesary, but we'll find out in CI.